contracts/action-handlers/PayRewardTokensHandler.sol

```diff
@@ -21,22 +22,26 @@ contract PayRewardTokensHandler is Ownable, Versionable {
   address public actionDispatcher;
   address public tokenManagerAddress;
   address public rewardPoolAddress;
+  address public prepaidCardManagerAddress;
```

Comment:
This is changing the order of the storage variables - the new variable must go at the end